src/codegen/browser/test.ts

```diff
@@ -150,6 +150,23 @@ function buildBrowserNodeGraph(events: BrowserEvent[]) {
           },
         }
 
+      case 'form-submitted':
+        return {
+          type: 'click',
+          nodeId: event.eventId,
+          button: 'left',
+          modifiers: {
+            ctrl: false,
+            shift: false,
+            alt: false,
+            meta: false,
+          },
+          inputs: {
+            previous,
+            locator: getLocator(event.tab, event.submitter),
+          },
+        }
```

Comment:
We don't need a specific primitive/action to submit a form. We can just do a click on the submit button instead.